bittensor/core/extrinsics/serving.py

```diff
@@ -253,6 +254,7 @@ def publish_metadata(
         period (Optional[int]): The number of blocks during which the transaction will remain valid after it's submitted. If
             the transaction is not included in a block within that number of blocks, it will expire and be rejected.
             You can think of it as an expiration date for the transaction.
+        reset_bonds (bool): If ``True``, the function will reset the bonds for the neuron. Defaults to ``False``.
```

Comment:
```suggestion
        reset_bonds (bool): If `True`, the function will reset the bonds for the neuron. Defaults to `False`.
```
We are trying to get rid of double brackets. There will be a deep refactoring in version 10. But for now we are trying not to introduce new ones.